anipose/common.py

```diff
@@ -191,6 +191,7 @@ def get_calibration_board(config):
 
     board_size = calib['board_size']
     board_type = calib['board_type'].lower()
+    manually_verify = calib['manually_verify']
```

Comment:
This will fail if the "manually_verify" option is not specified, breaking people's current configs.
It should default to false. To do this, plase add `'manually_verify': False` within the `DEFAULT_CONFIG` variable in `anipose/anipose.py`